nodes/store/data.js

```diff
@@ -69,13 +69,13 @@ const setters = {
             const filtered = []
             for (const m of msg) {
                 if (canSaveInStore(base, node, m)) {
-                    filtered.push(m)
+                    filtered.push(structuredClone(m))
```

Comment:
`structuredClone` requires NodeJS 17+ 

but dashboard engine specifies `"node": ">=14"`

since we have access to the `RED` object, we should probably use `RED.util.cloneMessage`

```suggestion
                    filtered.push(config.RED.util.cloneMessage(m))
```